README.md

```diff
@@ -67,6 +67,7 @@ managed by Docker or Kubernetes.
 This is useful when testing or for development instances or when a load-balancer mandates a non-standard port.
 * `LOG_FORMAT_NAME` - Can be set to `text` or `json` (default).
 * `NO_LOGGING_URL_PARAMS` - Can be set to `TRUE` if you don't want to log url params. Default is empty which means URL params are logged
+* `NO_LOGGING_BODY` - Defaults to true `TRUE`.  Set otherwise and nginx should log the request_body.
```

Comment:
Would be great to change it to `LOG_BODY=FALSE`, but I see that `NO_LOGGING_URL_PARAMS` is there already. This is similar to double negatives in English, which is very confusing :-) /cc @timgent